src/index.css

```diff
@@ -0,0 +1,4 @@
+.new-category-title {
```

Comment:
Rather than making a class, let's do this with Bootstrap spacing utility classes. It saves code and makes it easier to tell in the JSX how we're styling it.